Update renotification rules for incident setting (#23022)
DataDog/documentation: content/en/service_management/incident_management/incident_settings.md

2. Under **For incidents matching...**, select the incident property field `key:value` pairs you want notifications to be sent for. By default, these filters are empty, and a notification rule triggers for any incident.
3. **Notify**: Select your notification recipients. Notifications can be sent to any of Datadog's existing [notification integrations][18]. If you want to notify a recipient's mobile device, select the option for their name that includes **(Mobile Push Notification)**. The recipient must have enabled notifications in the [Datadog mobile app][19] for this option to appear.
4. **With Template**: Select the desired message template you want the notification rule to use.

> 5. **Renotify on updates to**: Select the incident properties that will trigger renotifications. A new notification is sent whenever one or more of the selected properties change. **Note**: properties already in your filters (see step 2) are automatically included in the renotification rules.

Reviewer: ```suggestion
5. **Renotify on updates to**: Select the incident properties that trigger renotifications. A new notification is sent whenever one or more of the selected properties change. **Note**: properties already in your filters (see step 2) are automatically included in the renotification rules.
```

Author: ```suggestion
5. **Renotify on updates to**: Select the incident properties that trigger notifications. A new notification is sent whenever one or more of the selected properties change. **Note**: properties already in your filters (see step 2) are automatically included in these rules.
```
